src/vm/codeman.cpp

```diff
@@ -4300,7 +4318,7 @@ LPCWSTR ExecutionManager::GetJitName()
     
     if (NULL == pwzJitName)
     {
-        pwzJitName = MAKEDLLNAME_W(W("clrjit"));
+        pwzJitName = const_cast<LPWSTR>(MAKEDLLNAME_W(W("clrjit")));
```

Comment:
It was constant in my original change. If you see the commit I linked to in the description, @pgavlin changed it to  LPWSTR because CLRConfig::GetConfigValue expects it to be LPWSTR.

The conflict is between MAKEDLLNAME returning LPCWSTR vs GetConfigValue expecting it to be LPWSTR.